### PR TITLE
Created method to add user to an organization

### PR DESCRIPTION
**Pull request by daconex**

### CODE REVIEW

File: lib/Github/Api/Organization/Members.php [hunk 1]
```diff
@@ -46,6 +46,15 @@ public function conceal($organization, $username)
         return $this->delete('orgs/'.rawurlencode($organization).'/public_members/'.rawurlencode($username));
     }
 
+    /*
+     * Add user to organization
+     */
+
```
Comment (GrahamCampbell) [inline lib/Github/Api/Organization/Members.php]: cs

Comment (daconex) [inline lib/Github/Api/Organization/Members.php]: ?

Comment (GrahamCampbell) [inline lib/Github/Api/Organization/Members.php]: no extra newline after the phpdoc, and no extra space character on the end of the function definition

**Comment by GrahamCampbell** [comment]
Also, this is missing test coverage too. :trollface:

**Comment by daconex** [comment]
Developing php on Windows :frowning:

**Comment by cursedcoder** [comment]
Please squash at least 2 last commits, I don't think someone is very interested with commit message "Ugh"

**Comment by daconex** [comment]
@cursedcoder Done. I squashed all the changes into I single commit!

**Comment by cursedcoder** [comment]
Thanks

**Comment by Hadrien45** [comment]
Thanks

**Pull request merged**